TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -202,13 +202,16 @@ public static void setNext(CommandContext cmd, CommandSender sender) {
         if (cmd.argsLength() > 0) {
             MapContainer found = null;
             for (MapContainer mapContainer : TGM.get().getMatchManager().getMapLibrary().getMaps()) {
-                if (mapContainer.getMapInfo().getName().equalsIgnoreCase(cmd.getJoinedStrings(0))) {
+                if (mapContainer.getMapInfo().getName().equalsIgnoreCase(cmd.getJoinedStrings(0).toLowerCase())) {
```

Comment:
Don't need .toLowerCase() on the command since you're using .equalsIgnoreCase()

Reply:
oops, removed it